tornjak-frontend/src/components/agent-create-join-token.tsx

```diff
@@ -146,7 +152,7 @@ class CreateJoinToken extends Component<CreateJoinTokenProp, CreateJoinTokenStat
 
     axios.post(endpoint, cjtData)
       .then(res => this.setState({ message: "Request:" + JSON.stringify(cjtData, null, ' ') + "\n\nSuccess:" + JSON.stringify(res.data, null, ' ') }))
-      .catch(err => displayResponseError("Agent creation failed.", err))
+      .catch(err => showResponseToast(err))
```

Comment:
Since we are not doing the error trace here, it would be a good idea to specify that error happened during the Agent create process?